File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int z, o, t;
        z = o = t = 0;
        for (int i = 0; i < nums.size(); i++) {
            if (nums[i] == 0) z++;
            if (nums[i] == 1) o++;
            if (nums[i] == 2) t++;
        }
        int i = 0, c = 0;
        for (; i < z; i++)
            nums[c++] = 0;
        i = 0;
        for (; i < o; i++)
            nums[c++] = 1;
        i = 0;
        for (; i < t; i++)
            nums[c++] = 2;
    }
};

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        z, o, t = 0, 0, 0
        for i in nums:
            if i == 0: z += 1
            elif i == 1: o += 1
            else: t += 1
        i = 0
        while z > 0:
            nums[i] = 0
            z -= 1
            i += 1
        while o > 0:
            nums[i] = 1
            o -= 1
            i += 1
        while t > 0:
            nums[i] = 2
            t -= 1
            i += 1
        